login/login.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://edithui.netlify.app/components/usage.css">
+    <link rel="stylesheet" href="login.css">
+    <title>Login</title>
+</head>
+<body>
+    <section class="login-container">
+        <div class="login-card">
+            <h1 class="login-title">Login</h1>
+            <div class="input-container">
+                <label for="userName">UserName*</label>
+                <input class="input input-primary" type="text" placeholder="Type Username" id="userName">
+                <label for="Password">Password*</label>
+                <input class="input input-primary" type="text" placeholder="Type Password" id="password">
```

Comment:
Use attribute type "password" on the password input field.

Reply:
Done